File: app/src/main/java/com/example/mykasir/feature_manajemen_produk/model/Product.kt
package com.example.mykasir.feature_manajemen_produk.model

data class Product(
    val id: Long = System.currentTimeMillis(),
    val name: String,
    val category: String = "",
    val price: Int = 0,
    val stock: Int = 0,
    val minStock: Int = 0,
    val imageUri: String? = null
)

File: BackEnd_MyKasir/models/Product.js
const { DataTypes } = require("sequelize");
const sequelize = require("../config/database");

const Product = sequelize.define(
  "Product",
  {
    id: {
      type: DataTypes.INTEGER,
      primaryKey: true,
      autoIncrement: true,
    },
    name: {
      type: DataTypes.STRING,
      allowNull: false,
    },
    category: {
      type: DataTypes.STRING,
      defaultValue: "",
    },
    price: {
      type: DataTypes.INTEGER,
      allowNull: false,
      defaultValue: 0,
    },
    stock: {
      type: DataTypes.INTEGER,
      allowNull: false,
      defaultValue: 0,
    },
    minStock: {
      type: DataTypes.INTEGER,
      defaultValue: 0,
    },
    imageUri: {
      type: DataTypes.STRING,
      allowNull: true,
    },
  },
  {
    tableName: "products",
    timestamps: true,
  }
);

module.exports = Product;
